schemas/public/onpremises-kfd-v1alpha2.json

```
@@ -1580,11 +1625,11 @@
       "properties": {
         "podCidr": {
           "$ref": "#/$defs/Types.Cidr",
-          "description": "The pod cidr to use for the cilium pods"
+          "description": "FIXME: The CIDR for the Pods network. Set if must be different from `.spec.kubernetes.podCidr`."
```

@nutellinoit may be you have some insight on this

---

This is used only when cilium is used and permits to use a different cidr from the cidr defined in the kubernetes cluster, since we are using the cluster cilium mode. It's not used in the case calico is choosen.

If empty, the kubernetes one will be used

---

so, it's like an "override" for the `.spec.kubernetes.podCidr`, right?

Would you say that the proposed description (without the `FIXME:` part) is correct?

---

What do you think about this @nutellinoit?:
```suggestion
          "description": "Allows specifing a CIDR for the Pods network different from `.spec.kubernetes.podCidr`. If not set the default is to use `.spec.kubernetes.podCidr`."
```